docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -1,33 +1,38 @@
 # Getting Started with EdgeX Snaps
 
+[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgexfoundry)
+
+This folder contains snap packaging for the EdgeX Snap
+
+The snap currently supports both `amd64` and `arm64` platforms.
+
 ## Introduction
+
 [Snaps](https://snapcraft.io/docs) are a hybrid of traditional Linux packages and containers. Snaps can be installed on any
 Linux distro with snap support enabled, although full confinement currently requires some flavor of Ubuntu (Desktop/Server
 or [Core](https://ubuntu.com/core/docs)).
 
 Canonical publishes snaps (amd64 & arm64) for each release of EdgeX to the Snap Store. In contrast to docker deployment, all core,
-security, support, and runtime dependencies are provided in a single snap called [edgexfoundry](https://snapcraft.io/edgexfoundry).
-Additional snaps are available for App Service Configurable (https://snapcraft.io/edgex-app-service-configurable), as well as the
-standard set of EdgeX reference device services (see list below). The edgexfoundry snap also includes Device Virtual to allow users
+security, support, and runtime dependencies are provided in a single snap called edgexfoundry.
```

Comment:
Please revert to have the hyperlink: `[edgexfoundry](https://snapcraft.io/edgexfoundry)`

Reply:
Reverted!